docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,138 @@
+---
+id: running-and-editing-timelines
+title: Running and Editing Timelines
+sidebar_label: Running and Editing Timelines
+---
+
+
+Running a timeline to timecode
+------------------
+
+The HTP (intensity) of fixtures in the timeline is controlled by the fader level of the handle. The console will automatically fire the timeline playback to full when timecode is received within the range of the timeline settings (you can turn this off using the option [Autoload in Range](../timelines/timeline-options.md#timecode-tab)).
```

Comment:
It has now been renamed "Activate in range" (#87417) to avoid confusion with Cuelist autoloading